### PR TITLE
SME & peer review remarks for RHOARDOC-1140

### PR DESCRIPTION
**Pull request by tradej**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

Follow-up on #783.

### CODE REVIEW

File: docs/howto/logging-to-a-file/index.adoc [hunk 1]
```diff
@@ -44,14 +44,14 @@ Under `logging`, add a `periodic-rotating-file-handlers` section with a new hand
 periodic-rotating-file-handlers:
   FILE:
     file:
-      path: target/MY_APP.log
+      path: target/MY_APP_NAME.log
```
Comment (tradej) [inline docs/howto/logging-to-a-file/index.adoc:55]: I did this to be consistent with the fabric8-launcher docs.

**Review by Ladicek — DISMISSED** [review]

File: docs/howto/logging-to-a-file/index.adoc [hunk 2]
```diff
@@ -44,14 +44,14 @@ Under `logging`, add a `periodic-rotating-file-handlers` section with a new hand
 periodic-rotating-file-handlers:
```
Comment (Ladicek) [inline docs/howto/logging-to-a-file/index.adoc:52]: Comment to line 27: adding a formatter isn't strictly necessary, there are two pre-defined formatters available. One called `PATTERN`, suitable for logging to a file, and `COLOR_PATTERN`, suitable for logging to an interactive console. Showing how to create your own pattern is certainly desirable, though, so I wouldn't remove that -- I'd just mention that there are 2 pre-defined patterns available (or 1, really, when it comes to logging to a file).

**Comment by Ladicek** [comment]
Otherwise LGTM.

File: docs/howto/logging-to-a-file/index.adoc (hunk 2 above)
Comment (tradej) [reply docs/howto/logging-to-a-file/index.adoc:52]: OK, will do, thanks.

**Review by bobmcwhirter — APPROVED** [review]

**Pull request merged**